src/main/java/au/org/sports/wrugby/scoreboard/ScorerController.java

```diff
@@ -32,6 +32,51 @@
     @Value("${au.org.sports.wrugby.scoreboard.logFile}")
     public String fileName;
 
+    @Value("${au.org.sports.wrugby.scoreboard.preset1.quarter-length}")
```

Comment:
There's a lot of values here.  Perhaps this would be better in a PresetConfig class.  You can then create multiple configurations in a separate bean and just wire them in here:

@Configuration
public class PresetConfiguration() {
    @Value("${au.org.sports.wrugby.scoreboard.preset1.quarter-length}")
    public String preset1QuarterLength;

    // put the reset of the value wiring here as well

    @Bean(name = "preset1")
    public Preset preset1 {
        return new Preset(preset1QuarterLength, preset1ShotClockLength, preset1DisplayShotClock, preset1PlayerTimeouts, preset1CoachTimeouts);
    }

    @Bean(name = "preset2")
    public Preset preset2 {
        return new Preset(preset2QuarterLength, preset2ShotClockLength, preset2DisplayShotClock, preset2PlayerTimeouts, preset2CoachTimeouts);
    }

    @Bean(name = "preset3")
    public Preset preset3 {
        return new Preset(preset3QuarterLength, preset3ShotClockLength, preset3DisplayShotClock, preset3PlayerTimeouts, preset3CoachTimeouts);
    }
}

In the ScorerController, you should now be able to autowire in the three presets using @Qualifier of preset1, preset2 and preset3 accordingly.  This should make the ScorerController a bit simpler.